man/systemd-analyze.xml

```diff
@@ -1183,6 +1189,23 @@ boot  efi  exitrd  init  lib32	libx32	media	    opt  root  sbin  sys  usr  vmlin
 
       <xi:include href="version-info.xml" xpointer="v258"/>
     </refsect2>
+
+    <refsect2>
+      <title><command>systemd-analyze unit-gdb <replaceable>SERVICE</replaceable></command></title>
+
+      <para>Spawn and attach a debugger to the given service. By default,
+      <citerefentry project='man-pages'><refentrytitle>gdb</refentrytitle><manvolnum>1</manvolnum></citerefentry>
+      will be used. This may be changed using the <option>--debugger=</option> option or the
+      <varname>$SYSTEMD_DEBUGGER</varname> environment variable. Use the <option>--debugger-arguments=</option> option
+      to pass extra command line arguments to the debugger and quote as appropriate
+      when <replaceable>ARGS</replaceable> contain whitespace (See Example).</para>
+
+      <example>
+        <programlisting>$ systemd-analyze -A="-batch -ex 'info all-registers'" unit-gdb systemd-oomd.service</programlisting>
```

Comment:
hmm short arguments do not take `=`? on long arguments do. hence change -A= to --debugger-arguments= here?